### DISCUSSION TITLE
Disable a single Button

### DISCUSSION THREAD

**Discussion by itsNiccoloSabatini** [emilk/egui#4263] [Q&A] [answered]
there is any way to disable a single button in EGUI, when i use ui.set_enable(false) it disable every interactable widget i define, any way i can do it but in one single button?

**Comment by emilk** [comment, marked as answer]
There's `ui.add_enabled` (tip: it's mentioned in the docs for `set_enabled`!)

**Reply by itsNiccoloSabatini** [reply]
ty so much, i didn't notice it in the Docs.
